spec.md

```diff
@@ -118,3 +142,16 @@ The chunk format is as follows:
 - Subquantizers: `quantized_len` x (`reconstructed_len` / `quantized_len`) x `sizeof(quantized_type)`
 - Norms: `matrix_rows` x `sizeof(reconstructed_type)`
 - Quantized embedding matrix: `matrix_rows` x `quantized_len` x `sizeof(reconstructed_type)`
+
+### Metadata
+
+- Chunk identifier: 5
+- UTF8 encoded Metadata: `chunk_data_len` times `u8`
```

Comment:
Maybe add that the metadata is expected to be in TOML format.

Reply:
done